ServerCore/Pages/Teams/AddMember.cshtml.cs

```diff
@@ -34,12 +34,7 @@ public async Task<IActionResult> OnGetAsync(int teamId)
                 return NotFound("Could not find team with ID '" + teamId + "'. Check to make sure you're accessing this page in the context of a team.");
             }
 
-            Users = await (from user in _context.PuzzleUsers
```

Comment:
What's the query this generates?

Reply:
I don't know how to tell. Can we look at this together?

Reply:
Generated queries show up in the debugger output when the query runs. Happy to look at this with you as well

Reply:
Current:

```
SELECT [p].[ID], [p].[Email], [p].[EmployeeAlias], [p].[IdentityUserId], [p].[IsGlobalAdmin], [p].[MayBeAdminOrAuthor], [p].[Name], [p].[PhoneNumber], [p].[TShirtSize], [p].[VisibleToOthers]
FROM [PuzzleUsers] AS [p]
WHERE NOT (EXISTS (
    SELECT 1
    FROM [EventAdmins] AS [e]
    INNER JOIN [Events] AS [e0] ON [e].[EventID] = [e0].[ID]
    INNER JOIN [PuzzleUsers] AS [p0] ON [e].[AdminID] = [p0].[ID]
    WHERE ([e0].[ID] = @__entity_equality_Event_0_ID) AND ([p0].[ID] = [p].[ID])))
```

Proposed: (Note: the INNER JOIN is completely unnecessary and easily removed by doing an EventID equality test instead of an Event equality test; I don't know why Entity Framework is doing this)

```
SELECT [e].[AdminID]
FROM [EventAdmins] AS [e]
INNER JOIN [Events] AS [e0] ON [e].[EventID] = [e0].[ID]
WHERE [e0].[ID] = @__entity_equality_thisEvent_0_ID
ORDER BY [e].[AdminID]

SELECT [p].[ID], [p].[Name], [p].[Email]
FROM [PuzzleUsers] AS [p]
ORDER BY [p].[ID]
```

Reply:
Proposed after next commit:

```
SELECT [e].[AdminID]
FROM [EventAdmins] AS [e]
WHERE [e].[EventID] = @__thisEvent_ID_0
ORDER BY [e].[AdminID]

SELECT [p].[ID], [p].[Name], [p].[Email]
FROM [PuzzleUsers] AS [p]
ORDER BY [p].[ID]
```